docs/source/user_guide/calculation_plugins/cp.rst

```diff
@@ -48,6 +47,8 @@ Inputs
 
   Those keywords should not be specified, otherwise the submission will fail.
 
+  The `SYSTEM`, `ibrav` keyword is optional. If it is not specified, `ibrav=0` is used. When a non-zero `ibrav` is given, `aiida-quantumespresso` automatically extracts the cell parameters. As a consistency check, the cell is re-constructed from these parameters and compared to the input cell. The input structure needs to match the convention detailed in the `pw.x documentation <https://www.quantum-espresso.org/Doc/INPUT_PW.html#idm199>`_. The tolerance used in this check can be adjusted with the `IBRAV_CELL_TOLERANCE` key in the `settings` dictionary. It defines the absolute tolerance on each element of the cell matrix.
```

Comment:
Just for my understanding, the consistency check on the parameters is done by the `get_parameters_from_cell` function as well, correct?

Reply:
Exactly, that's here: https://github.com/aiidateam/qe-tools/blob/develop/qe_tools/converters/_structure.py#L68-L72